MapboxVision/Services/MotionManager.swift

```diff
@@ -4,28 +4,25 @@ import CoreLocation
 
 final class MotionManager {
     private let motion: CMMotionManager = CMMotionManager()
-    private let referenceFrame: CMAttitudeReferenceFrame
+    private let referenceFrame: CMAttitudeReferenceFrame = .xArbitraryZVertical
```

Comment:
Please move it to high-level making it static const.

Reply:
done